course1/module7/Question2.js

```diff
@@ -0,0 +1,30 @@
+/*2. Write a function filterObj that will filter out all the keys of a flat object that have objects or arrays using Object.keys and Object.entries.
+ Example:
+let obj = {
+ a:”Apple”,
+ b:[“Basketball”,”Baseball”],
+ c: {
+  call: “cellphone”
+ },
+ d: “Dog”
+}
+filterObject(obj) //This should return {a:”Apple”, d:”Dog”}
+ */
+
+let obj = {
+    a:"Apple",
+    b:["Basketball","Baseball"],
+    c: {
+     call: "cellphone"
+    },
+    d: "Dog"
+   }
+
+   const arrayOfEntries = Object.entries(obj);  //Returns an array of key/values of the enumerable properties of an object
```

Comment:
Can you write similar logic using object.keys aswell?

Reply:
Yes.